app/components/ui/create-tale-modal/component.js

```diff
@@ -150,14 +150,12 @@ export default Component.extend({
     const self = this;
     self.set("creatingTale", false);
     self.handleError(e);
-    $('.ui.modal.compose-error').modal('show');
     $('.ui.modal.compose-error').modal({
```

Comment:
```suggestion
    $('#compose-error-modal').modal({
```
This should fix the duplicate modal issue for the create-tale / AiWT error cases